packages/amplify-category-analytics/src/utils/pinpoint-helper.ts

```diff
@@ -1,12 +1,20 @@
 import {
   AmplifySupportedService,
   pathManager, readCFNTemplate,
-  open, $TSAny, $TSContext, $TSMeta, AmplifyCategories,
+  open, $TSAny, $TSContext, $TSMeta, AmplifyCategories, stateManager,
 } from 'amplify-cli-core';
 import { printer } from 'amplify-prompts';
 import * as path from 'path';
 import { getAnalyticsResources } from './analytics-helper';
 
+/**
+ *
+ */
+export type PinpointApp = {
```

Comment:
For future discussions.

we should tweak eslint to enforce docs only on types that we export outside of single package.

enforcing docs for internals isn't very productive.

Reply:
I actually missed it, it was added on save, but will leave it for now to not dismiss the approval, unless the recent changes broke something and will need tweaked